File: 800/1676B/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() 
{
    int t;
    cin >> t;  // number of test cases

    while (t--) 
    {
        int n;
        cin >> n;  // number of children

        vector<int> a(n);
        cin >> a[0];
        int mini = a[0];  // track minimum candies

        // read remaining candies and find minimum
        for (int i = 1; i < n; i++) 
        {
            cin >> a[i];
            mini = min(mini, a[i]);
        }

        // compute total moves required
        int sum = 0;
        for (int i = 0; i < n; i++) 
        {
            sum += a[i] - mini;
        }

        cout << sum << "\n";
    }

    return 0;
}

File: 800/1676B/main.java
import java.util.*;
public class main
{
    public static void main(String args[])
    {
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();
        
        while(t-- > 0)
        {
            int n = sc.nextInt();
            int a[] = new int[n];
            
            a[0] = sc.nextInt();
            int mini = a[0];
            
            for(int i=1;i<n;i++)
            {
                a[i] = sc.nextInt();
                mini = Math.min(mini, a[i]);
            }
            
            int sum = 0;
            for(int i=0;i<n;i++)
                sum += a[i] - mini;
            
            System.out.println(sum);
        }
    }
}
